.github/workflows/linux-browser-compatibility-test.yml

```diff
@@ -0,0 +1,27 @@
+name: Linux Browser Compatibility Test
+
+on:
+  pull_request:
+      branches:
+        - main
+        - release-1.x
+
+env:
+  SELENIUM_GRID_PROVIDER: saucelabs
+  CLOUD_WATCH_METRIC: false
+  TEST_TYPE: Browser-Compatibility-Test
+  SAUCE_USERNAME: ${{secrets.SAUCE_USERNAME}}
+  SAUCE_ACCESS_KEY: ${{secrets.SAUCE_ACCESS_KEY}}
+
+jobs:
+  call-setup-action-host:
+    uses: ./setup-action-host.yml
```

Comment:
If there's no specific reason, I would like to keep them in one `.yml` file.

Is `setup-action-host.yml` reused anywhere?

Reply:
This is no longer there